lib/go-util/ptr.go

```diff
@@ -21,6 +21,10 @@ package util
 
 import "time"
 
+func Ptr[T any](v T) *T {
+	return &v
+}
+
```

Comment:
This PR adds several additional usages of `util.StrPtr()` and `util.IntPtr()` that can be replaced with `util.Ptr()` now that it exists